todo_CLI.js

```diff
@@ -0,0 +1,142 @@
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const fs = require('fs');
+
+function readTodoList(){
+  //expect an array return
+  if(fs.existsSync('todo.json') === true){
```

Comment:
The condition should have been "execute read file if a json file is provided in the command line".